crp_APL/planners/planner_base/include/planner_base/wrapperBase.hpp

```diff
@@ -13,7 +13,7 @@
 #include <tf2/LinearMath/Quaternion.h>
 #include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
 
-#include "plannerInterfaces/plannerInterfaces.hpp"
+#include "../../../interfaces/plannerInterfaces/plannerInterfaces.hpp"
```

Comment:
Should we prefer includes with relative path to the source file in planners/APL? (currently the interface folder is included in cmake)

Reply:
I better like the keep it simple includes such as **#include "plannerInterfaces/plannerInterfaces.hpp"**

Reply:
I had errors with build, if include directory is used, therefore I kept the original code until further bugfix